hack/ginkgo-e2e.sh

```diff
@@ -99,6 +99,11 @@ elif [[ "${KUBERNETES_PROVIDER}" == "aws" ]]; then
   auth_config=(
     "--auth_config=${HOME}/.kubernetes_auth"
   )
+elif [[ "${KUBERNETES_PROVIDER}" == "conformance_test" ]]; then
```

Comment:
@jlowdermilk 
So, in ginkgo-e2e, skip the following if AUTH_CONFIG is set:
  - `source "${KUBE_VERSION_ROOT}/cluster/${KUBERNETES_PROVIDER}/util.sh"`
  - `prepare-e2e`
  - `detect-master`